### ISSUE TITLE
Cursor not moving

### ISSUE THREAD

**Issue by oSumAtrIX** [DDDinggo22/Osu-Bot#8]
Offset0 : 48c
Offset1 : 27c
Offset2 : f8
Offset3 : 7b0
Offset4 : 70
ThreadOffset : -32C

Tried with this offset from another solution.

**Comment by TheCodingNina** [comment]
I am sorry for my absence. I recently took some time off programming, because I got somewhat busy with somethings IRL.
After that I started learning how to use DirectX to improve the GUI elements for my (next/new) projects.

Right now, I am finishing up the last stages before I can start testing Osu!Bot V3 for the first time. This will require me to find a new time pointer, so that gives me a good excuse to make a detailed video tutorial on how to find it yourself (if I forget to update it.)

Also I have setup a discord server so that I can give some more information on the projects progression. Feel free to join:  [Disguard](discord.me/Disguard).

**Comment by blulere** [comment]
I have the same problem. All that happens is my mouse pointer gets limited to the osu! window, if I try to move it out, it teleports back. If I have selected the osu!bot window, the mouse cursor gets stuck and I have to use my tablet to get it on the osu! window. The bot dosen't actually do any 'auto-ing', and I can't find any source of help. The closest things I found are demos and people showing their own osu!bots.
